src/PAModel/Serializers/SourceSerializer.cs

```diff
@@ -375,9 +375,18 @@ private static void LoadSourceFiles(CanvasDocument app, DirectoryReader director
             }
 
             // When loading TestSuites sharded files, add them within the top parent AppTest control (i.e. Test_7F478737223C4B69)
+            // Make sure to load the the Test_7F478737223C4B69.fx.yaml file first to add the top parent control.
```

Comment:
Is there an assumption about control ordering here? Why does the order of AddControl matter?

Reply:
Earlier we had dissembled AppTest control and sharded each of the TestSuite inside it into individual files. Yesterday while investigating one of the telemetry bugs, I got hold of an app which had some properties defined at the root level, and since we were not persisting the top parent those properties were getting lost.

With this change we are still sharding TestStuies into individual files but we are still keeping the top parent (Test_7F478737223C4B69) file as well to persist root level properties if any.

When adding control we have to add the top parent control first so that we can append the children to it, hence this handling.

Reply:
Could we add a case that demonstrates this to sourcebaseline tests?

Reply:
sure will do.